language-configuration-erb.json

```diff
@@ -0,0 +1,31 @@
+{
+	"comments": {
+		"blockComment": [ "<%#", "%>" ]
+	},
+	"brackets": [
+		["{", "}"],
+		["[", "]"],
+		["(", ")"],
+		["<", ">"]
+	],
+	"autoClosingPairs": [
+		["{", "}"],
+		["[", "]"],
+		["(", ")"],
+		["`", "`"],
+		{ "open": "'", "close": "'", "notIn": ["string", "comment"] },
+		{ "open": "\"", "close": "\"", "notIn": ["string"] },
+		["<", ">"],
+		{ "open": "%", "close": "%", "notIn": ["string", "comment"] }
+	],
+	"surroundingPairs": [
+		["{", "}"],
+		["[", "]"],
+		["(", ")"],
+		["\"", "\""],
+		["'", "'"],
+		["`", "`"],
+		["<", ">"],
+		["%", "%"]
```

Comment:
I think the last one (just the erb tags) is the best of those you've offered. I'll have a look and do some testing later.